README.md

```diff
@@ -1,8 +1,6 @@
 # fast-json-stringify&nbsp;&nbsp;[![Build Status](https://travis-ci.org/fastify/fast-json-stringify.svg?branch=master)](https://travis-ci.org/fastify/fast-json-stringify)
 
-__fast-json-stringify__ is two times faster than `JSON.stringify()`.
-It is particularly suited if you are sending small JSON payloads, the
-advantages reduces on large payloads.
+__fast-json-stringify__ is significantly faster than `JSON.stringify()` for small payloads. Its performance advantage shrinks as your payload grows.
```

Comment:
Can you explain change a bit more? Have you experience a degradation in throughput?

Reply:
I was trying to just rephrase the existing idea based on the benchmarks. The most significant change was removing "two times faster." I thought it was clearer to focus on the performance characteristics instead of using a rough average of the performance improvement that might not apply to many users.

Reply:
👍  ok.